lib/openstack_query/runners/user_runner.py

```diff
@@ -25,14 +24,6 @@ class UserRunner(QueryRunner):
 
     DEFAULT_DOMAIN = UserDomains.STFC
 
-    @staticmethod
-    def _get_marker(obj: User) -> str:
-        """
-        This method returns the marker value for a User object, required for pagination
-        :param obj: An Server object to get the marker property for
-        """
-        return UserProperties.get_prop_mapping("id")(obj)
-
```

Comment:
Why has this method been removed?

Reply:
Looks to be a redundant function that I should have deleted